sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb

```diff
@@ -314,6 +314,19 @@
     ")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "pass-arguments"
+   },
+   "outputs": [],
+   "source": [
+    "#passing the fixed settings or parameters\n",
+    "arguments = { 'early_stopping':True, 'evaluation_frequency': 2}\n",
+    "image_object_detection_job.set_image_model(**arguments)"
```

Comment:
In dpv2, we have a dedicated function set_image_model for this which accepts a fixed set of parameters.
Because of this, I think it is better to do some thing like .set_image_model(early_stopping=True, evaluation_frequency=) instead of using a arguments dictionary. Could you please update this

Reply:
Also, could you please use evaluation_frequency=1 ?

Reply:
Maybe we can use evaluation_frequency=1 similar to how we have it in cli examples.